test/apify/storages/request_queue_reset.ts

```diff
@@ -0,0 +1,32 @@
+// $ npm run build && ts-node test/apify/storages/request_queue_reset.ts
+
+import { Actor } from 'apify';
+import log from '@apify/log';
+import { purgeLocalStorage } from 'crawlee';
+import { CheerioCrawler } from '@crawlee/cheerio';
+
+// RequestQueue auto-reset when stuck with requests in progress
+Actor.main(async () => {
+    process.env.APIFY_INTERNAL_TIMEOUT = '30000'; // 30s
```

Comment:
the `APIFY_INTERNAL_TIMEOUT` env var should be renamed to `CRAWLEE_INTERNAL_TIMEOUT`